### DISCUSSION TITLE
Excuse me, time tags are crowded together, how to maintain spacing?

### DISCUSSION THREAD

**Discussion by ApaulMyLittleAirport** [Live-Charts/LiveCharts2#777] [Q&A]
![image](https://user-images.githubusercontent.com/45100828/204088177-42f4c347-a05e-4bb0-8f82-943186ac6b67.png)

**Comment by beto-rodriguez** [comment]
Hi, this seems like an issue, can please see your code?

as a work around you can rotate the labels as in this sample: https://lvcharts.com/docs/WPF/2.0.0-beta.330/samples.axes.labelsFormat

**Reply by ApaulMyLittleAirport** [reply]
Thank you very much for taking time out of your busy schedule to reply to my question. When I set “LabelsRotation = 15”, It looks fine.
My code is pretty simple:

Avalonia  11.0.0-preview4
LiveChartsCore.SkiaSharpView.Avalonia   2.0.0-beta.502-11.0.0-preview2

        public Axis[] XAxes { get; set; } =
        {
        new Axis
        {


       Labeler = value => new DateTime((long) value).ToString("MM-dd HH:mm:ss"),
            LabelsRotation = 0,
             LabelsPaint = new SolidColorPaint
            {
                Color = SKColors.White,

                FontFamily = LiveChartsSkiaSharp.MatchChar('汉')

            },
            
            UnitWidth = TimeSpan.FromDays(10).Ticks, 

            MinStep = TimeSpan.FromDays(1).Ticks
        }
        };

<lvc:CartesianChart Name="LChart"  Series="{Binding Series,Mode=TwoWay}" LegendBackground="Transparent" LegendTextBrush="White" LegendPosition="Top" ZoomMode="X"  XAxes="{Binding XAxes}"/>
